app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/adapters/ProfileCourseListRecyclerAdapter.kt

```diff
@@ -29,8 +29,8 @@ class ProfileCourseListRecyclerAdapter internal constructor(
 
         val course: CourseData? = courses[position]
         val myDashboardViewHolder = holder as MyDashboardViewHolder
-        myDashboardViewHolder.tvCourseTitle.text = course?.course_title
-        holder.itemView.setOnClickListener { profileCourseListFragment.openCourseDetail(courses[position].course_slug) }
+        myDashboardViewHolder.tvCourseTitle.text = course?.courseTitle
+        holder.itemView.setOnClickListener { profileCourseListFragment.openCourseDetail(courses[position].courseTitle) }
```

Comment:
courseTitle -> should be courseSlug instead I think.

Reply:
Possible. I'll check it out

Reply:
I am unable to open course details page from profile as we are passing title instead of slug.